Sources/Transaction/Inputs/DefaultMixinSelectionStrategy.swift

```diff
@@ -4,16 +4,21 @@
 
 import Foundation
 
-final class DefaultMixinSelectionStrategy: MixinSelectionStrategy {
+final class DefaultMixinSelectionStrategy: IdempotentMixinSelectionStragegy {
```

Comment:
for backwards compatibility lets keep this file as-it-was, and make a new selection strategy implementation that inherits from `IdempotentMixinSelectionStragegy`

Reply:
ok actually I think I have better idea. will test out in a separate branch.

Reply:
actually, I think your solution is correct, users get existing behavior from the extension